src/main/java/ui/GUIController.java

```diff
@@ -41,6 +44,7 @@ public GUIController(UI ui, PanelControl panelControl, Label apiBox) {
 
     public final void registerEvents() {
         UI.events.registerEvent((ModelUpdatedEventHandler) this::modelUpdated);
+        UI.events.registerEvent((UpdateRateLimitsEventHandler) this::updateSyncRefreshRate);
```

Comment:
Feels like it would be neater to have a single event handler for this which then dispatches to the two methods.